nuxt.config.js

```diff
@@ -49,7 +49,16 @@ export default {
   /*
    ** Global CSS
    */
-  css: ['@brown-ccv/disco-styles'],
+  css: [
```

Comment:
I think this should be just the `disco.scss` since that imports the others (not tested theory)